src/compiler/scala/tools/nsc/transform/patmat/Solving.scala

```diff
@@ -510,14 +510,14 @@ trait Solving extends Logic {
 
     type TseitinSearch = List[(Array[Clause], List[Lit])]
 
-    /** An implementation of the DPLL algorithm for checking statisfiability
+    /** An implementation of the DPLL algorithm for checking satisfiability
       * of a Boolean formula in CNF (conjunctive normal form).
       *
       * This is a backtracking, depth-first algorithm, which searches a
       * (conceptual) decision tree the nodes of which represent assignments
       * of truth values to variables. The algorithm works like so:
       *
-      * - If there are any empty clauses, the formula is unsatisifable.
+      * - If there are any empty clauses, the formula is unsatisfiable.
```

Comment:
non-satisfiable

Reply:
insatiable

Reply:
![](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTQhyGrvtFcVbk3zMt2B4DQWM0Wz3_z6ZmQ6rP07MJSneFRGrje&s)

Reply:
The negation was also used in the sense of "seeking satisfaction": "The more notorious the offence, the more unsatisfiable the injury."